recipe/patches/0011-Reapply-CUDA-HIP-Add-a-__device__-version-of-std-__g.patch

```diff
@@ -1,82 +1,61 @@
-From 9b725e564cdc58771a4f98b54d3f03d0af20481d Mon Sep 17 00:00:00 2001
```

Comment:
This is not a patch.

Reply:
Ah, nevermind, just saw your
> Sorry, I am on a slowish network [...]

in the main thread.